src/components/shared/FilterNavigatorBar.css

```diff
@@ -59,8 +60,8 @@
 }
 
 .filterNavigatorBarLeafItem {
-  padding-right: 8px;
-  margin-right: -8px;
+  /* padding-right: 8px;
+  margin-right: -8px; */
```

Comment:
Please don't leave the lines commented. If you want to remove it, you can just remove the line.